sdk/python/kfp/compiler/compiler.py

```diff
@@ -868,7 +868,7 @@ def _write_workflow(workflow: Dict[Text, Any], package_path: Text = None):
         will be returned.
     """
     yaml.Dumper.ignore_aliases = lambda *args : True
-    yaml_text = yaml.dump(workflow, default_flow_style=False)
+    yaml_text = yaml.dump(workflow, default_flow_style=False, default_style='>')
```

Comment:
Would it be OK to use `default_style='I'`?
It works with multiline strings too.

Reply:
Sorry, this was a typo. I meant the pipe character" `|`